Automattic/pocket-casts-ios
# Show gravatar

Based on an user feedback I give it a try on adding Gravatar avatar under Profile.

| Non Plus user | Expiring Plus user | Plus user |
| ------------ | ------------------ | ---------- |
| ![Simulator Screen Shot - iPhone 14 - 2022-12-07 at 13 18 27](https://user-images.githubusercontent.com/7040243/206239016-65c480c7-9b91-4e53-847a-f4ee581d1b93.png) | ![Simulator Screen Shot - iPhone 14 - 2022-12-07 at 13 17 03](https://user-images.githubusercontent.com/7040243/206239505-e11f6923-1c1f-4bfa-8a8b-b30865c0c014.png) | ![Simulator Screen Shot - iPhone 14 - 2022-12-07 at 13 17 30](https://user-images.githubusercontent.com/7040243/206239532-1d92709b-14d2-4508-8b6f-d2deb1366c44.png) |

## To test

1. Create an account in gravatar.com and set your avatar (if you don't have one)
2. Run the app
3. Login with your email
4. Check the Profile tab
5. ✅ It should show your Gravatar
6. Logout
7. ✅ The avatar should disappear
7. Login with an account without gravatar
8. ✅ The app won't show any gravatar avatar and it will fallback to the way it was before

## Checklist
